charts/oes/charts/spinnaker/templates/hooks/install-using-hal.yaml

```diff
@@ -147,3 +157,18 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: status.hostIP
+      - command:
+        - bash
+        - /tmp/config/spin-pipeline-import.sh
```

Comment:
What does this script do?

Reply:
This script pulls the sample pipleines from git and saves them to spinnaker